applications/nrf_desktop/doc/main.rst

```diff
@@ -0,0 +1,29 @@
+.. _nrf_desktop_main:
+
+Main module
+###########
+
+The main module is the entry point of the nRF Desktop application.
+
+
+Module Events
+*************
+
++------------------+-------------+--------------------------+------------------------+------------------+
+| Source Module    | Input Event | This Module              | Output Event           | Sink Module      |
++==================+=============+==========================+========================+==================+
+|                  |             |  :ref:`nrf_desktop_main` | ``module_state_event`` |                  |
++------------------+-------------+--------------------------+------------------------+------------------+
+
+
+Configuration
+*************
+
+The module has no configuration options.
+
+
+Implementation details
+**********************
+
+
+The :ref:`nrf_desktop_main` module initializes the :ref:`event_manager` and then submits the module state event to inform the other modules that they can continue the application startup.
```

Comment:
Should we change it to `module_state_event` for consistency?

Reply:
Done